assesspy/load_data.py

```diff
@@ -1,29 +1,52 @@
-# Import necessary libraries
+from importlib.resources import as_file, files
+
```

Comment:
This is apparently the hot new way to import package data. `setuptools` was complaining to me about `pkg_resources` being deprecated.